web/src/main/java/no/nav/ung/sak/web/app/tjenester/saksbehandler/SaksbehandlerRestTjeneste.java

```diff
@@ -12,21 +12,19 @@
 import jakarta.ws.rs.Produces;
 import jakarta.ws.rs.QueryParam;
 import jakarta.ws.rs.core.MediaType;
+import no.nav.k9.felles.integrasjon.microsoftgraph.MicrosoftGraphTjeneste;
```

Comment:
Ser ut som den sliter med å injecte MicrosoftGraphTjeneste inn i SaksbehandlerRestTjeneste.

`org.jboss.weld.exceptions.DeploymentException: WELD-001408: Unsatisfied dependencies for type MicrosoftGraphTjeneste with qualifiers @Default
  at injection point [BackedAnnotatedParameter] Parameter 1 of [BackedAnnotatedConstructor] @Inject public no.nav.ung.sak.web.app.tjenester.saksbehandler.SaksbehandlerRestTjeneste(MicrosoftGraphTjeneste, @KonfigVerdi String, HistorikkRepository, BehandlingRepository)
  at no.nav.ung.sak.web.app.tjenester.saksbehandler.SaksbehandlerRestTjeneste.<init>(SaksbehandlerRestTjeneste.java:0)`

Reply:
Takk :+1:  Jeg ser jeg har glemt å legge til beans.xml-fila i modulen i k9-felles..